### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Crissov**
at → in, bysupervisor → betreut von, urlalso → auch verfügbar unter

### CODE REVIEW

**Comment by DavidLuptak** [comment]
Please consider the newly created [Translation Guideline](https://github.com/michal-h21/biblatex-iso690/wiki/Translation-Guideline) as mentioned also in referencing issue #49 .
If everything is all right, just leave a comment here and I can merge the pull request right away.

File: german-iso.lbx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+\ProvidesFile{german-iso.lbx}
+[2016/05/15 v0.3.1 biblatex localization]
+
+\InheritBibliographyExtras{german}
+\InheritBibliographyStrings{german}
+
+\DeclareBibliographyStrings{%
+  at           = {{in}% maybe 'bei'
```
Comment (Crissov) [inline german-iso.lbx]: `in` is definitely the wrong translation, but I’m not sure which one would be correct (`Ort`, `Exemplar`, …)

**Comment by Crissov** [comment]
- `at` → _Standort_
- `bysupervisor` dubious, but _betreut von_ matches other roles like `translator` → _übersetzt von_
- `urlalso` → _auch verfügbar unter_

File: german-iso.lbx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                  {in}},
+  bysupervisor = {{betreut von}% maybe Betreuer, but gender-dependent then, female: Betreuerin
+                  {betreut von}},
```
Comment (DavidLuptak) [inline german-iso.lbx]: Exactly, there is no explicit section about academic theses in the standard, it should be read between the lines..

**Comment by DavidLuptak** [comment]
Thanks @Crissov for updating the https://github.com/michal-h21/biblatex-iso690/wiki/Translation-Guideline#translation-matrix!

**Pull request merged**